src/api/critter/critter.swagger.ts

```diff
@@ -171,7 +171,7 @@ export const critterPaths = {
       requestBody: {
         content: {
           'application/json': {
-            schema: CritterCreateSchema
+            schema: BulkCritterCreateSchema
```

Comment:
The other schema changes seem to just be naming changes, except for this one, which swaps out the non-bulk schema for a the bulk one. Is that intentional? Looks like they are mostly the same, except that the original schema doesn't allow both itis tsn and scientific name to be provided, whereas the bulk one does.

Reply:
(Looking at the payloads from SIMS, the bulk create schema does line up with what sims sends. So it is working.)